api/objects.proto

```diff
@@ -249,6 +249,14 @@ message Cluster {
 	// If the key is empty, the node will be unlocked (will not require a key
 	// to start up from a shut down state).
 	repeated EncryptionKey unlock_keys = 9;
+	
+	// RoutingMeshOff denotes whether the routing mesh is currently disabled.
+	// By default the routing mesh is enabled. As result of user removing the
+	// ingress network, this flag will be set to true. Cluster needs to carry
+	// this information, so that on reboot or leader re-election, new leader
+	// will not automatically create the routing mesh if it cannot find one
+	// in store. 
+	bool routingMeshOff = 10;
```

Comment:
I did not opt for `enabled/disabled` because this variable indicates a state, which could be just temporary until user creates a custom ingress network, I thought `on/off` was more appropriate for a state while `enabled/disabled` is more for a config (which we do not have).

But not being a native speaker, `routing_mesh_disabled` works just as fine for me. Will change to that.